src/System.Text.JsonLab/System/Text/Json/Utf8JsonReader.cs

```diff
@@ -40,6 +36,8 @@ public int MaxDepth
             }
         }
 
+        public bool Instrument { get; set; }
```

Comment:
Does it need to be public?

Reply:
I believe so. How else would the caller indicate they want line number/position info to be tracked?